packages/data-point-cache/lib/cache.js

```diff
@@ -39,15 +53,28 @@ function getFromStore (cache, key) {
 function get (cache, key) {
   return cache.redis.exists(key).then(exists => {
     if (exists) {
-      return getFromStore(cache, key)
+      return module.exports.getFromStore(cache, key)
```

Comment:
Can I ask you why "module.exports." has been added?

Reply:
this is so i can spy on the method on testing